netci.groovy

```diff
@@ -22,7 +22,7 @@ def osGroupMap = ['Ubuntu':'Linux',
                   'RHEL7.2': 'Linux']
 
 // Map of osName -> nuget runtime
-def targetNugetRuntimeMap = ['OSX' : 'osx.10.10-x64',
+def targetNugetRuntimeMap = ['OSX' : 'osx.10.12-x64',
```

Comment:
Yes, the setMachineAffinity call being the critical one here (otherwise it will run on 10.11)